src/main/java/millom/sandbox/service/WeatherService.java

```diff
@@ -0,0 +1,26 @@
+package millom.sandbox.service;
+
+import millom.sandbox.mapper.NasaMapper;
+import millom.sandbox.pojos.Weather;
+
+public class WeatherService {
+  MyClientService myClientService = new MyClientService();
```

Comment:
default access modifier? Is it on purpose or?

Reply:
changed to private access modifier.

```
 private final MyClientService myClientService;
  private final NasaMapper nasaMapper;
```